controllers/ipfscluster_controller.go

```diff
@@ -136,21 +136,26 @@ func (r *IpfsClusterReconciler) createTrackedObjects(
 ) map[client.Object]controllerutil.MutateFn {
 	sa := corev1.ServiceAccount{}
 	svc := corev1.Service{}
+	gwsvc := corev1.Service{}
+	apisvc := corev1.Service{}
 	cmScripts := corev1.ConfigMap{}
 	secConfig := corev1.Secret{}
 	sts := appsv1.StatefulSet{}
 
 	mutsa := r.serviceAccount(instance, &sa)
 	mutsvc, svcName := r.serviceCluster(instance, &svc)
 
+	mutgwsvc, _ := r.serviceGateway(instance, &gwsvc)
+	mutapisvc, _ := r.serviceAPI(instance, &apisvc)
+
```

Comment:
Should we consider moving this under the below if condition on [line 170](https://github.com/ipfs-cluster/ipfs-operator/pull/74/files#diff-e38a482631026e4366d3147b3f02a2f45f3d1661d89d40cdefbd98157208204aR170)?